docs/SearchSpaceSpec.md

```diff
@@ -67,8 +72,19 @@ The candidate type and value for variable is here:
    * Suitable for a discrete variable with respect to which the objective is smooth and gets smoother with the size of the variable, which is bounded from one side.
 <br/>
 
-Note that SMAC only supports a subset of the types above, including `choice`, `randint`, `uniform`, `loguniform`, `quniform(q=1)`. In the current version, SMAC does not support cascaded search space (i.e., conditional variable in SMAC).
 
-Note that GridSearch Tuner only supports a subset of the types above, including `choic`, `quniform` and `qloguniform`, where q here specifies the number of values that will be sampled. Details about the last two type as follows
+## Relation With Tuners
```

Comment:
looks good. could you change the title to "Search space types supported by each Tuner".
Then, please list all the tuners in the table below.